packages/deployments/contracts/contracts/interfaces/IAavePool.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.11;
+
+interface IAavePool {
```

Comment:
do aave pools support wrapping and unwrapping?

Reply:
Are you referring to wrapping/unwrapping native assets? At the protocol level, Aave only handles ERC20. They have[ this helper contract](https://github.com/aave/aave-v3-periphery/blob/ca184e5278bcbc10d28c3dbbc604041d7cfac50b/contracts/misc/WETHGateway.sol#L42-L49) to allow users to wrap/unwrap and enter/exit the pool in one transaction

Reply:
basically want a way to get the underlying asset instead of `aTokens` so you dont have to go through an AMM and the user doesn't need to know a loan was taken out. is there a way to do that?